common/components/common/projects/EditProjectForm.jsx

```diff
@@ -0,0 +1,188 @@
+// @flow
+
+import React from 'react';
+import type {LinkInfo} from '../../../components/forms/LinkInfo.jsx'
+import type {FileInfo} from '../../common/FileInfo.jsx'
+import ImageUploadFormElement from '../../../components/forms/ImageUploadFormElement.jsx'
+import LinkList from '../../../components/forms/LinkList.jsx'
+import FileUploadList from '../../../components/forms/FileUploadList.jsx'
+import TagCategory from '../../common/tags/TagCategory.jsx'
+import TagSelect from '../../common/tags/TagSelect.jsx'
+import DjangoCSRFToken from 'django-react-csrftoken'
+import ProjectAPIUtils from '../../../components/utils/ProjectAPIUtils.js';
+import type {ProjectDetailsAPIData} from '../../../components/utils/ProjectAPIUtils.js';
+import _ from 'lodash'
```

Comment:
i'd recommend importing this as `lodash` because it's a JS idiom to reserve `_` for denoting unused variables, like

```
try {
  foo()
} catch (_) {
  console.error('something bad happened')
}
```

Reply:
'_.' is also the idiomatic way to use the lodash/underscore libraries, and helpful for brevity.